virttest/qemu_devices/qdevices.py

```diff
@@ -907,23 +907,56 @@ def hotplug_qmp(self):
         if params.get("host-nodes"):
             host_nodes = list(map(int, params["host-nodes"].split()))
             params["host-nodes"] = host_nodes
+        for k in params:
+            if params[k] == "yes":
+                params[k] = True
+            if params[k] == "no":
+                params[k] = False
         kwargs = {"qom-type": backend,
                   "id": params.pop("id"),
                   "props": dict(params)}
         return "object-add", kwargs
 
     def verify_unplug(self, out, monitor):
```

Comment:
what's the param `out` used for ?

Reply:
@yanan-fu  I checked the whole file,  all `verify_unplug()` functions have same parameters. This param seems useless in this function, but if we want to remove it, we need to update all the other functions, and where they are called.  Let's don't rock the boat. Thanks!

Reply:
@huangyum   A long-standing problem, no effect for our testing but better to solve it. To avoid new function call with the same method, it is up to you.
Anyway, it is not related with this patch, current code is ok for me :)